reviews/tuf-graduation.md

```diff
@@ -0,0 +1,64 @@
+# TUF Graduation Application
+
+The Update Framework (TUF) helps developers maintain the security of a software update system, even against attackers that compromise some repository or signing keys. TUF provides a flexible framework and specification that developers can adopt into any software update system.
+
+## TUF fulfills all the incubating and graduation criteria:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+* TUF is used in production by Cloudflare, Datadog, DigitalOcean, Docker, IBM, Kolide, Microsoft, RedHat, VMware, and many others.  [Uptane](https://uptane.github.io/), the automotive variant of TUF, is sold by major automotive vendors and is being adopted by automakers with over 1/3 of the cars on US roads in their forthcoming cars.  Uptane is included as part of [Automotive Grade Linux](https://www.automotivelinux.org/), another Linux Foundation project.
```

Comment:
Is it the specification, or the reference implementation thereof that is used in production by these adopters?  I think the answer to that is probably contained somewhere in the links under https://theupdateframework.github.io/adoptions.html, but it would be good to summarize the answer here.

Reply:
Added in https://github.com/cncf/toc/pull/166/commits/822588a8bf4a2da22972ba12cdd6ce790df08c40